core/src/test/java/io/questdb/cutlass/line/tcp/LineTcpReceiverTest.java

```diff
@@ -291,6 +291,7 @@ public void testFirstRowIsCancelled() throws Exception {
         runInContext((receiver) -> {
             send(receiver, "table", WAIT_ENGINE_TABLE_RELEASE, () -> {
                 try (LineTcpSender lineTcpSender = new LineTcpSender(Net.parseIPv4("127.0.0.1"), bindPort, msgBufferSize)) {
+                    lineTcpSender.disableValidation();
```

Comment:
some of these tests do nasty stuff (like mangled column names, etc) we don't want regular users to do. so we have to disable validations.